plasma_framework/test/src/framework/utils/ExitPriority.test.js

```diff
@@ -116,8 +116,8 @@ contract('ExitPriority', () => {
             expect(parsedExitableAt).to.be.bignumber.equal(new BN(exitableAt));
         });
 
```

Comment:
This is more of a general comment towards exit priority tests. Can we add a function to `ExitPriority` that returns the tx position from the exit priority and assert in all tests that compute the exit priority that all 3 input parameters are the same than what is extracted from the exit priority number.